README.md

```diff
@@ -38,7 +38,40 @@ at `/config` inside the container. It will be copied into place prior to buildin
 Note that `make olddefconfig` will be run regardless to ensure the latest
 options have been applied.
 
-## Reproducibile builds
+## Building kernels in Qubes
+
+Here's how to set up a build environment in [Qubes], suitable for use with [SecureDrop].
+The build requires `docker`, so make sure your TemplateVM has docker configured.
```

Comment:
Could you also link to `#enabling-grsecurity-patches` and mention that you'll need to export a few environment variables with the grsec credentials. I think the `#enabling-grsecurity-patches` section also might need updating to include the `GRSECURITY` env var.

Reply:
The `#enabling-grsecurity-patches` section should probably mention that you need to make a request for these credentials if you don't have them and to make sure to store them securely in your password manager.